src/main/java/no/fint/betaling/controller/FileController.java

```diff
@@ -20,12 +20,15 @@
 @RequestMapping(value = "/api/file")
 public class FileController {
 
-    @Autowired
     private FileService fileService;
 
-    @Autowired
     private GroupService groupService;
 
+    public FileController(FileService fileService, GroupService groupService) {
+        this.fileService = fileService;
+        this.groupService = groupService;
+    }
+
     @PostMapping
     public ResponseEntity getCustomersOnFile(@RequestHeader(name = "x-school-org-id") String schoolId, @RequestBody byte[] file) throws NoVISIDColumnException, UnableToReadFileException, InsufficientDataException, HttpMediaTypeNotAcceptableException {
```

Comment:
Burde egentlig `x-school-org-id` vært en del av api url'en?

Reply:
This is set as a TODO in the next update.